internal/gitsync/github/handler_test.go

```diff
@@ -0,0 +1,81 @@
+package github
```

Comment:
These test cases are honestly pretty bad, I basically wrote them just for the coverage. There's not really any higher-level properties to test here, just checking that it handles errors sanely and calls methods on its dependencies as expected.